www/src/components/shell/TerminalSidebar.js

```diff
@@ -473,7 +506,7 @@ function Step3() {
       <CodeLine marginTop="medium">
         plural deploy --commit "your message"
       </CodeLine>
-      <P
```

Comment:
I have changed some `P` to `Div` as it was giving an error in the console since `div` cannot be a child of `p`.